### PR TITLE
Feature: Integrate Translator Service

### PR DESCRIPTION
**Pull request by ay0503**
Added code to let the client request make a REST API request to the translator service.

### CODE REVIEW

**Review by lexik04 — COMMENTED** [review]
this looks great to me!

File: public/src/client/topic.js [hunk 1]
```diff
@@ -71,12 +71,28 @@ define('forum/topic', [
...
 
+		function configurePostToggle() {
+			$(".topic").on("click", ".view-translated-btn", function () {
+					// Toggle the visibility of the next .translated-content div
+					$(this).closest('.sensitive-content-message').next('.translated-content').toggle();
+					// Optionally, change the button text based on visibility
+					var isVisible = $(this).closest('.sensitive-content-message').next('.translated-content').is(':visible');
+					if (isVisible) {
+							$(this).text('Hide the translated message.');
+					} else {
```
Comment (KevnKey) [inline public/src/client/topic.js:89]: Looks like it shows the translated message without hiding the original, seems like a good way to do it

**Review by KevnKey — COMMENTED** [review]
This looks great overall, the separate folder for translate definitely keeps the modularity

**Review by vsolskyyy — COMMENTED** [review]
Very well organized changes and code. Maybe you could add comments to explain the code, but otherwise good job.

**Pull request merged**